docs/data-ai/capture-data/capture-sync.md

```diff
@@ -705,25 +678,101 @@ To view all the captured data you have access to, go to the [**DATA** tab](https
+### Direct to MongoDB capture
 
-{{< readfile "/static/include/data/capture-supported.md" >}}
+You can configure direct capture of tabular data to a MongoDB instance alongside disk storage on your edge device.
+This can be useful for powering real-time dashboards before data is synced from the edge to the cloud.
+
+Configure using `mongo_capture_config` attributes in your data manager service.
+
+Here is a sample configuration that will capture fake sensor readings both to the configured MongoDB URI as well as to the `~/.viam/capture` directory on disk:
+
+{{< expand "Click to view configuration" >}}
+
+```json
+{
+  "components": [
+    {
+      "name": "sensor-1",
+      "namespace": "rdk",
+      "type": "sensor",
+      "model": "fake",
+      "attributes": {},
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "method": "Readings",
+                "capture_frequency_hz": 0.5,
+                "additional_params": {}
+              }
+            ]
+          }
+        }
+      ]
+    }
+  ],
+  "services": [
+    {
+      "name": "data_manager-1",
+      "namespace": "rdk",
+      "type": "data_manager",
+      "attributes": {
+        "mongo_capture_config": {
+          "uri": "mongodb://127.0.0.1:27017/?directConnection=true&serverSelectionTimeoutMS=2000"
+        }
+      }
+    }
+  ]
+}
+```
+
+{{< /expand >}}
+
+When `mongo_capture_config.uri` is configured, data capture will attempt to connect to the configured MongoDB server and write captured tabular data to the configured `mongo_capture_config.database` and `mongo_capture_config.collection` (or their defaults if unconfigured) after enqueuing that data to be written to disk.
+
+If writes to MongoDB fail for any reason, data capture will log an error for each failed write and continue capturing.
+
+Failing to write to MongoDB doesn't affect capturing and syncing data to cloud storage other than adding capture latency.
+
+{{< alert title="Caution" color="caution" >}}
+
+- Capturing directly to MongoDB may write data to MongoDB that later fails to be written to disk (and therefore never gets synced to cloud storage).
+- Capturing directly to MongoDB does not retry failed writes to MongoDB. As a consequence, it is NOT guaranteed all data captured will be written to MongoDB.
+  This can happen in cases such as MongoDB being inaccessible to `viam-server` or writes timing out.
+- Capturing directly to MongoDB may reduce the maximum frequency that data capture can capture data due to the added latency of writing to MongoDB.
+  If your use case needs to support very high capture rates, this feature may not be appropriate.
+
+{{< /alert >}}
+
+### Conditional sync
+
+Although data sync checks for new data at the configured interval, you can define additional conditions that must be met to trigger syncing.
```

Comment:
`Although` doesn't make sense here. There's no contradiction in that - rather it works on top of this: 

```suggestion
By default, `viam-server` checks for new data to sync at the configured interval (`sync_interval_mins `).
You can additionally configure sync to only happen when certain conditions are met.
```